spec/requests/api/users_spec.rb

```diff
@@ -129,4 +129,70 @@
       end
     end
   end
+
+  describe 'DELETE /api/users/:id' do
+    subject do
+      delete("/api/users/#{target_id}", headers:)
+      response
+    end
+
+    let!(:current_user) { create(:user, :admin) }
+    let!(:target_user) { create(:user) }
+
+    context 'アクセストークンがある場合' do
+      let(:headers) { { 'Authorization' => "Bearer #{access_token}" } }
+
+      context 'アクセストークンが有効期限内の場合' do
+        let(:access_token) { AccessToken.new(email: current_user.email).encode }
+
+        context '自分以外のユーザIDの場合' do
+          context 'ユーザが存在する場合' do
+            let(:target_id) { target_user.id }
+
+            it '204でユーザが削除される' do
```

Comment:
もうちょっと日本語を補いたいですね。
```suggestion
            it 'ユーザが削除され、204を返す' do
```

「204で」だけだと「204を使って」などの他の解釈ができそうなので、ちゃんと記述をしたほうが誤解が少ないです。

また、書き直しの際に気づいたのですが

* 削除をして
* 最後に 204 を返す

という処理なので、説明順序も逆にしたほうがわかりやすいですね。

Reply:
他の箇所も同じくですね。